### PR TITLE
Jdy33

### PR DESCRIPTION
**Pull request by rusefillc**

### CODE REVIEW

File: firmware/console/binary/tunerstudio_io_serial.cpp [hunk 1]
```diff
@@ -2,19 +2,186 @@
...
+int SerialTsChannel::start(uint32_t baud) {
 	SerialConfig cfg = {
 		.speed = baud,
 		.cr1 = 0,
 		.cr2 = USART_CR2_STOP1_BITS | USART_CR2_LINEN,
 		.cr3 = 0
 	};
 
-	sdStart(m_driver, &cfg);
+	if (1) {
```
Comment (mck1117) [inline firmware/console/binary/tunerstudio_io_serial.cpp:60]: `if (1)`?

Comment (rusefillc) [reply firmware/console/binary/tunerstudio_io_serial.cpp:60]: @dron0gus tell me how to adjust

Comment (mck1117) [reply firmware/console/binary/tunerstudio_io_serial.cpp:60]: @dron0gus poke?

**Comment by dron0gus** [comment]
Re-created PR https://github.com/mck1117/wideband/pull/171

**Pull request closed without merging**